mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx

```diff
@@ -58,11 +60,14 @@ interface MentorRequestsSectionProps {
  * Section with requests to become Way mentor
  */
 export const MentorRequestsSection = (props: MentorRequestsSectionProps) => {
+  const {language} = languageStore;
+  const mentorsRequestCollection = LanguageService.way.ComponentMentorRequestsSection;
```

Comment:
use LanguageService... in the code without creating const

Reply:
delete const mentorsRequestCollection